src/components/calcite-block-section/calcite-block-section.e2e.ts

```diff
@@ -131,12 +135,14 @@ describe("calcite-block-section", () => {
     expect(toggle.getAttribute("aria-label")).toBe(TEXT.expand);
 
     await toggle.click();
+    await page.waitForChanges();
```

Comment:
@driskull Added this but am getting the error from the next check. Note that if I comment out that check, the check for "open" also fails.